packages/web-shared/components/Searchbar/Search.styles.js

```diff
@@ -84,12 +84,14 @@ const Wrapper = withTheme(styled.div`
     border-radius: 0;
   }
 
-  .chip-list-feature {
-    * > &:first-child {
+  .chip-list-feature ul {
+    overflow-x: scroll;
```

Comment:
```suggestion
    overflow-x: auto;
```

So that the empty scrollbar doesn't show up if we can't scroll.

![2024-02-13 11 35 20](https://github.com/ApollosProject/apollos-embeds/assets/72768221/2af375e3-542c-45f5-a06f-92d27fe7a800)

Reply:
dig it